app/controllers/api_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class ApiController < ApplicationController
```

Comment:
This controller should not inherit from ApplicationController.
We use two types of controllers in Rails, one for rendering "views", for example, when we make a website with Rails, and the other one, is for APIs.
We are creating an API one here, so the `ApiController` should inherit from `ActionController::API`.
Finally we should have two "types", `ApplicationController`, which inherit from ActionController::Base, and `ApiController`, which inherit from `ActionController::API`.

Reply:
fixing this